### ISSUE TITLE
 Expected RBRACE unknown property grid-template-columns

### ISSUE THREAD

**Issue by pauledd** [CSSLint/csslint#753]
Hi
using CSSLint (2.0.22) with brackets.
Having a simple css file with:

```
#content {
	display: grid;
	grid-template-columns: repeat(4, 1fr);
}
```
getting this error:
```
1	
Disallow IDs in selectors - Don't use IDs in selectors. (ids)	#content {
3	
Require use of known properties - Unknown property 'grid-template-columns'. (known-properties)	grid-template-columns: repeat(4, 1fr);
3	
Parsing Errors - Expected RBRACE at line 3, col 35. (errors)	grid-template-columns: repeat(4, 1fr);
```
Whats wrong with that?

**Comment by JaredTamana** [comment]
Also an issue here with many `grid` properties:
- `grid-template-areas`
- `grid-template-columns`
- `grid-template-rows`
- `grid-area`

Are these deprecated or something? Plugin version 2.0.22 from Brackets. 

EDIT: This is not an issue with CSSLint's main project. Something hasn't been merged to cfjedimaster/brackets-csslint. An issue should be raised there, and it should be an easy fix. This issue should now be closed.

**Comment by AWOL-TECH** [comment]
I also have an issue being flagged  with grid-template-columns I am using the line below:
`    grid-template-columns: repeat(auto-fit, minmax(250px, 1fr));`

and receive this message: 

> "Expected RBRACE at line 255, col 59." 

 

I am using CSSLint installed via NPM at version 1.0.5. Also use the Atom linter plugin too

```

npm list -g --depth=1
C:\Users\###\AppData\Roaming\npm
+-- csslint@1.0.5
| +-- clone@2.1.2
| `-- parserlib@1.1.1

```
also got these results via csslint.net

![image](https://user-images.githubusercontent.com/18701905/62875600-36a41380-bd1b-11e9-95d3-3543333ccba6.png)

**Comment by JaredTamana** [comment]
@timeassistant: Separate issue I think, `repeat` is not an exported property in `csslint.js`

**Comment by AWOL-TECH** [comment]
@JaredTheWolf  ah ok, it is valid CSS though, is it something that can be corrected?

**Comment by irvingzamora** [comment]
So is there a solution or a workaround for this? I am having the same issue
`ul.products{grid-template-columns:repeat(12,1fr)}`

**Comment by JaredTheWolf** [comment]
> So is there a solution or a workaround for this? I am having the same issue
> 
> `ul.products{grid-template-columns:repeat(12,1fr)}`

Per my last comment: 

This is not an issue with CSSLint's main project. Something hasn't been merged to cfjedimaster/brackets-csslint. An issue should be raised there, and it should be an easy fix. This issue should now be closed.

**Comment by AntonioHouTX** [comment]
I'm still having the issue as well with grid-template rows, grid-template-columns, and grid-gap.  Keep getting the 'unknown property' error message.
![grid-csslint](https://user-images.githubusercontent.com/39347117/116742126-8ed1ec00-a9bc-11eb-9925-92b7606c7177.jpg)

**Comment by JaredTamana** [comment]
@AntonioHouTX As previously mentioned, this is not an issue with CSSLint itself but rather the Brackets port of it. You may wish to raise the issue on that board instead.
